src/Nethermind/Nethermind.Merge.AuRa/AuRaMergePlugin.cs

```diff
@@ -73,6 +73,11 @@ protected override PostMergeBlockProducerFactory CreateBlockProducerFactory()
                 _blocksConfig,
                 _api.LogManager);
 
+        protected override IBlockFinalizationManager InitializeMergeFinilizationManager()
+        {
+            return new MergeFinalizationManager(_blockFinalizationManager, _api.FinalizationManager, _poSSwitcher);
```

Comment:
Shouldn't be AuraMergeFinalizationManager here?

Reply:
fixed